backend/src/v5/processors/teamspaces/projects/models/commons/tickets.js

```diff
@@ -157,7 +157,7 @@ const processExternalData = async (teamspace, project, model, ticketIds, data) =
 		if (groups.stillUsed.size) {
 			const stillUsed = Array.from(groups.stillUsed);
 			const existingGroups = await getGroupsByIds(teamspace, project, model, ticketId,
-				stillUsed, { _id: 1 });
+				stillUsed.map(stringToUUID), { _id: 1 });
```

Comment:
it works, but isnt there a unit test that has to be updated ?

Reply:
I guess there isn't because there's no failed test cases. but yes we should probably make sure there's a check in place for that..

I'll have a look 😢